### PR TITLE
Make analyzer connect and read

### PR DESCRIPTION
**Pull request by ngiachou**
# Initial implementation of crypto-analyzer-service

The main points of this current implementation are as follow:

- Implementation of sending a _POST_ request to the `coinbase-adapter-service` which initiates the subscription of a specific product to coinbase's websocket and starts populating **rethinkdb**
- Implementation of connecting and reading new changes made to the `cryptobot` database and in particular new _inserts_ at the `messages` table
- Printing output to console

### CODE REVIEW

File: crypto-analyzer/src/utils/adapter_request_helper.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import requests
+HOST = "http://coinbase_adapter:9000"
+subscribe_endpoint = HOST + "/api/market/subscribe"
```
Comment (pagidas) [inline crypto-analyzer/src/utils/adapter_request_helper.py]: I think a good addition would be to _inject_ `HOST` and `PORT` from a config file - they can change if `env` is different, i.e. `dev` `staging` `prod`.

Let's say you have a config file:
```shell
coinbase-adapter {
     host = "coinbase_adapter"
     host = {?COINBASE_ADAPTER_HOST} // this env variable can be optional, but has a default value

     port = 9000
     port = {?COINBASE_ADAPTER_PORT}
}
```
Then you'd need a config reader who will read those values from the confg and put them call them where you need them, instead of being hardcoded. Of course the config will be populated with the `env` variables if we use any in the docker-compose or else the default values will be in there.

Summing up:
- Look for a library to read values from a config file. It should also support `env` variables somehow, like the example I mentioned.
- Implement a config reader functionality to use where you need

P.S. if you find a way to do that with `coinbase-adapter` you can do the same with the `rethinkdb` -- taking host and port from the config file.

I think those would be good and not so over engineered additions. :)

**Review by pagidas — CHANGES_REQUESTED** [review]

File: crypto-analyzer/src/utils/adapter_request_helper.py (hunk 1 above)
Comment (ngiachou) [reply crypto-analyzer/src/utils/adapter_request_helper.py]: Great! I'm on it!

**Review by pagidas — APPROVED** [review]
Good to go! :) -- I'll let you merge it just to be on the safe side
